### PR TITLE
CB2-1054: Raise the Test Event on event bridge

### PR DESCRIPTION
**Pull request by ghost**
## Raise the Test Event on event bridge
send events to event bridge 
https://jira.dvsacloud.uk/browse/CB2-1054

## Checklist
- [ ] Code has been tested manually
- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Comment by tom-bjss** [comment]
Just noticed one of my test names is wrong, will update

**Review by danielchildsbjss — APPROVED** [review]

**Review by Ian-Hodges — APPROVED** [review]

File: tests/unit/send.test.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    it('GIVEN an issue with eventbridge WHEN 6 events are sent and 1 fails THEN the failure is in the response.', async () => {
+      const mTestResult: TestActivity[] = Array<TestActivity>(6);
+      mTestResult[0] = createTestResult(-1);
+      const mSendResponse: SendResponse = { SuccessCount: 5, FailCount: 1 };
+      await expect(sendEvents(mTestResult)).resolves.toEqual(mSendResponse);
+    });
+  });
+});
+
+function createTestResult(axels?: number): TestActivity {
```
Comment (premghinde) [inline tests/unit/send.test.ts:64]: Is this used?

Comment (tom-bjss) [reply tests/unit/send.test.ts:64]: mTestResult[0] = createTestResult(-1);

Comment (tom-bjss) [reply tests/unit/send.test.ts:64]: I did have it used in more places but after refactoring to use a list, I was able to drop most of them apart from the erroneous one

**Pull request merged**